src/main/java/Cuboid.java

```diff
@@ -1,7 +1,7 @@
 
 public class Cuboid extends Rectangle{
 
-    double height;
+    private double height;
 
     public Cuboid(double width, double length, double height) {
         super(width, length);
```

Comment:
1) Error details
Method call getHeight() returned a value of "0.0", but "1.0" was expected. Field with name [height] has value [0.0]. expected:<1.0> but was:<0.0>
Please fix your constructor. You need to compare your input parameter with 0, not this.height